competency-insights-ui/src/pages/Profile/ProfileDetails.js

```diff
@@ -4,79 +4,39 @@ import MailOutlineOutlinedIcon from '@mui/icons-material/MailOutlineOutlined';
 import CallOutlinedIcon from '@mui/icons-material/CallOutlined';
 import { Contribution } from "./components/Contribution";
 import { PermanentDrawerLeft } from "../../components/Layout/NavBar";
-import { SkeletonProfile } from "../../components/Layout/SkeletonProfile";
-import useDataFetching from "../../services/useDataFetching";
-import {useMsal} from "@azure/msal-react";
+import { useDataProvider } from '../../services/dataService';
+import UseDataFetching from "../../services/useDataFetching";
+
+export const ProfileDetails = () => {
 
-export const ProfileDetails = ({ emailAddress, name }) => {
-    const [user, setUser] = useState(null);
-    const [okrs, setOKR] = useState(null);
     const [categories, setCategories] = useState({});
-    const [category, setCategory] = useState("Blogs");
-    const { instance } = useMsal();
-    const { data: categoriesData, isLoading: categoriesIsLoading } = useDataFetching('Data/categories.json', instance);
+    const [category, setCategory] = useState("Blog");
```

Comment:
" useState("Blog") ", what do we mean by this ?

Reply:
Default state of "category" has been set to Blog.